src/Nethermind/Nethermind.Core/Specs/ChainSpecStyle/IChainSpecLoader.cs

```diff
@@ -21,5 +21,6 @@ namespace Nethermind.Core.Specs.ChainSpecStyle
     public interface IChainSpecLoader
     {
         ChainSpec Load(byte[] data);
+        ChainSpec Load(string jsonData);
     }
```

Comment:
i remember there was a reason for which I had to make it byte[] only... I do not remember the exact reason. I think that not every implementation could provide a string (do not know why)

Reply:
I can't think of anything that could go wrong here